src/Compilers/CSharp/Test/Semantic/Semantics/DeconstructionTests.cs

```diff
@@ -68,22 +69,36 @@ static void Main()
     {
         long x;
         string y;
-        (x, y) = new C();
+        /*<bind>*/(x, y) = new C()/*</bind>*/;
     }
 }
 ";
-            var comp = CreateStandardCompilation(source);
-            comp.VerifyDiagnostics(
-                // (8,18): error CS1061: 'C' does not contain a definition for 'Deconstruct' and no extension method 'Deconstruct' accepting a first argument of type 'C' could be found (are you missing a using directive or an assembly reference?)
-                //         (x, y) = new C();
-                Diagnostic(ErrorCode.ERR_NoSuchMemberOrExtension, "new C()").WithArguments("C", "Deconstruct").WithLocation(8, 18),
-                // (8,18): error CS8129: No Deconstruct instance or extension method was found for type 'C', with 2 out parameters.
-                //         (x, y) = new C();
-                Diagnostic(ErrorCode.ERR_MissingDeconstruct, "new C()").WithArguments("C", "2").WithLocation(8, 18)
-                );
-        }
+            string expectedOperationTree = @"
+IDeconstructionAssignmentExpression (OperationKind.DeconstructionAssignmentExpression, Type: ?, IsInvalid) (Syntax: '(x, y) = new C()')
+  Left: 
+    ITupleExpression (OperationKind.TupleExpression, Type: (System.Int64 x, System.String y)) (Syntax: '(x, y)')
+      Elements(2):
+          ILocalReferenceExpression: x (OperationKind.LocalReferenceExpression, Type: System.Int64) (Syntax: 'x')
+          ILocalReferenceExpression: y (OperationKind.LocalReferenceExpression, Type: System.String) (Syntax: 'y')
+  Right: 
+    IObjectCreationExpression (Constructor: C..ctor()) (OperationKind.ObjectCreationExpression, Type: C, IsInvalid) (Syntax: 'new C()')
+      Arguments(0)
+      Initializer: 
+        null
+";
+            var expectedDiagnostics = new DiagnosticDescription[] {
+                // CS1061: 'C' does not contain a definition for 'Deconstruct' and no extension method 'Deconstruct' accepting a first argument of type 'C' could be found (are you missing a using directive or an assembly reference?)
+                //         /*<bind>*/(x, y) = new C()/*</bind>*/;
+                Diagnostic(ErrorCode.ERR_NoSuchMemberOrExtension, "new C()").WithArguments("C", "Deconstruct").WithLocation(8, 28),
+                // CS8129: No suitable Deconstruct instance or extension method was found for type 'C', with 2 out parameters and a void return type.
+                //         /*<bind>*/(x, y) = new C()/*</bind>*/;
+                Diagnostic(ErrorCode.ERR_MissingDeconstruct, "new C()").WithArguments("C", "2").WithLocation(8, 28)
```

Comment:
The diagnostic comments no longer include the locations.

Reply:
Yes, there is a bug tracking that issue in the operation unit test generator.